tests/lib/eslint/flat-eslint.js

```diff
@@ -963,7 +994,7 @@ describe("FlatESLint", () => {
 
                 await assert.rejects(async () => {
                     await eslint.lintFiles(["node_modules"]);
-                }, /All files matched by 'node_modules\/\*\*\/\*.js' are ignored\./u);
+                }, /All files matched by 'node_modules' are ignored\./u);
```

Comment:
With this update, the reported glob patterns match what is passed on the command line instead of the interpolated patterns we used. This matches the behavior of `eslintrc` and seems preferable to me.